components/atoms/StatusBadge.js

```diff
@@ -5,12 +5,16 @@ import propTypes from 'prop-types'
  */
 
 export default function StatusBadge(props) {
+  if (props.hidden) return null
```

Comment:
My assumption is since activeAgreement drives this that perhaps there will be a use case around it in the future.
The logic around the status badge is a bit unusual.

By default the component displays if hidden isn't passed in